packages/jest-fake-timers/src/legacyFakeTimers.ts

```diff
@@ -29,38 +34,42 @@ type Timer = {
 };
 
 type TimerAPI = {
-  cancelAnimationFrame: FakeTimersGlobal['cancelAnimationFrame'];
+  cancelAnimationFrame: typeof globalThis.cancelAnimationFrame;
   clearImmediate: typeof globalThis.clearImmediate;
   clearInterval: typeof globalThis.clearInterval;
   clearTimeout: typeof globalThis.clearTimeout;
   nextTick: typeof process.nextTick;
-
-  requestAnimationFrame: FakeTimersGlobal['requestAnimationFrame'];
+  requestAnimationFrame: typeof globalThis.requestAnimationFrame;
   setImmediate: typeof globalThis.setImmediate;
   setInterval: typeof globalThis.setInterval;
   setTimeout: typeof globalThis.setTimeout;
 };
 
+type FakeTimerAPI = {
+  cancelAnimationFrame: Mock<FakeTimers['_fakeClearTimer']>;
+  clearImmediate: Mock<FakeTimers['_fakeClearImmediate']>;
+  clearInterval: Mock<FakeTimers['_fakeClearTimer']>;
+  clearTimeout: Mock<FakeTimers['_fakeClearTimer']>;
+  nextTick: Mock<FakeTimers['_fakeNextTick']>;
+  requestAnimationFrame: Mock<FakeTimers['_fakeRequestAnimationFrame']>;
+  setImmediate: Mock<FakeTimers['_fakeSetImmediate']>;
+  setInterval: Mock<FakeTimers['_fakeSetInterval']>;
+  setTimeout: Mock<FakeTimers['_fakeSetTimeout']>;
+};
+
 type TimerConfig<Ref> = {
   idToRef: (id: number) => Ref;
   refToId: (ref: Ref) => number | void;
 };
 
 const MS_IN_A_YEAR = 31536000000;
 
-type GlobalThis = typeof globalThis;
-
-interface FakeTimersGlobal extends GlobalThis {
-  cancelAnimationFrame: (handle: number) => void;
-  requestAnimationFrame: (callback: (time: number) => void) => number;
-}
```

Comment:
Might be these types were copied from TS, because they do not exist on `global` in Node. `globalThis` defines these props, so now this is safe to remove, right?